### PR TITLE
FIX: finalImage 2.0 -> generate_final_image

### PR DESCRIPTION
**Pull request by pwollinger**
PR to:

* Fix the finalImage bug 
* Change the file format to jpeg 
* Chaged the finalImage name to generate_final_cover

That's all for now folks!

### CODE REVIEW

**Comment by pwollinger** [comment]
@yfaria could you approve this PR? thx!

File: album_cover_generator.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import io
+import PIL
+from PIL import ImageFilter, Image
+
+def generate_final_cover(cover, coverFilename):
```
Comment (yfaria) [inline album_cover_generator.py]: The naming of "coverFilename" is different from the others.

File: album_cover_generator.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import io
+import PIL
```
Comment (yfaria) [inline album_cover_generator.py]: The relevant imports are explicit already, there is no need to import the whole library here

File: main.py [hunk 3]
```diff
@@ -38,29 +37,12 @@
                     print(f"Listening: {artist_name} - {track_name}")
 
             response = requests.get(current_track['item']['album']['images'][0]['url'])
-            album_blur = finalImage("album.png")
+            time.sleep(1)
+            album_blur = generate_final_cover(response.content, "album.jpeg")
```
Comment (yfaria) [inline main.py]: This function doesn't return an object, there is no need to put in a variable.

**Review by yfaria — DISMISSED** [review]

File: album_cover_generator.py (hunk 1 above)
Comment (pwollinger) [reply album_cover_generator.py]: Changed!

File: album_cover_generator.py (hunk 2 above)
Comment (pwollinger) [reply album_cover_generator.py]: Changed!

File: main.py (hunk 3 above)
Comment (pwollinger) [reply main.py]: Changed!

**Review by yfaria — APPROVED** [review]

**Pull request merged**